packages/alfa-aria/src/features/svg/a.ts

```diff
@@ -0,0 +1,25 @@
+import { Element, getClosest, hasAttribute, Node } from "@siteimprove/alfa-dom";
+import * as Roles from "../../roles";
+import { Any, Feature, Role } from "../../types";
+
+/**
+ * @see https://www.w3.org/TR/SVG2/linking.html#AElement
```

Comment:
We need to reference the place that defines the ARIA role mapping for each element. Unfortunately, I think the closest we can get is https://www.w3.org/TR/svg-aam/#mapping_role_table.